src/app/auth/auth-registration/auth-registration.component.ts

```diff
@@ -182,15 +182,26 @@ export class AuthRegistrationComponent implements OnInit {
             const nrAutoLoginTracer = this.newRelic.createTracer('auto login');
             this.authService
               .login({
-                email: this.user.email,
+                username: this.user.email,
                 password: this.confirmPassword
               })
               .subscribe(
-                async res => {
-                  nrAutoLoginTracer();
-                  this.storage.remove('unRegisteredDirectLink');
-                  const route = await this.switcherService.switchProgramAndNavigate(res.programs);
-                  this.showPopupMessages('shortMessage', 'Registration success!', route);
+                async globalRes => {
+                  this.authService.globalLogin({
+                    apikey: globalRes.apikey,
+                    service: 'LOGIN'
+                  }).subscribe(
+                  async res => {
+                    nrAutoLoginTracer();
+                    this.storage.remove('unRegisteredDirectLink');
+                    const route = await this.switcherService.switchProgramAndNavigate(res.programs);
```

Comment:
Global login will not return the programs, we need to get the programs from core POST auth

Reply:
method call `globalLogin` not sure why. it created before. but it calling core API /auth.json.

Reply:
@shawnm0705 I added comment in code and also i created a .md file in './docs/workflows/auth-workflows.md'. and mention in README.md file and also mention in code comments.

and i rename the `globalLogin` to `directLoginWithApikey`.